node/pkg/logscribe/app.go

```diff
@@ -0,0 +1,87 @@
+package logscribe
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"bisonai.com/orakl/node/pkg/db"
+	"bisonai.com/orakl/node/pkg/logscribe/api"
+	"bisonai.com/orakl/node/pkg/logscribe/utils"
+	"github.com/gofiber/fiber/v2"
+	"github.com/rs/zerolog/log"
+)
+
+const (
+	logsChannelSize             = 10_000
+	DefaultBulkLogsCopyInterval = 3 * time.Second
+)
```

Comment:
it's optional comment, but I'd like to encourage you not to use const.
try to reference fiber [locals](https://docs.gofiber.io/api/ctx#locals)

Reply:
Good point Nick. But `logsChannelSize` is a local variable and `DefaultBulkLogsCopyInterval` only referenced from tests. I'll use `Locals` for passing the `logsChannel` channel